src/formatters/stylish.ts

```diff
@@ -23,7 +23,7 @@
  * @author Sindre Sorhus
  */
 
-import chalk from 'chalk';
+import chalk = require('chalk');
```

Comment:
```suggestion
import * as chalk 'chalk';
```

Reply:
@P0lip StripAnsi requires the same kind "required" import but doesn't seem to like the `import *` syntax.

How about switching `esModuleInterop` flag to `true`? Would that mess something up?

Reply:
Don't worry about it. I've already updated both of these dependencies on develop :)

Reply:
>How about switching esModuleInterop flag to true? Would that mess something up?

@P0lip Yep, I've seen that. But not being a transpiling guru, I'd genuinely be interested in your standpoint on that.